src/components/datepicker/datepicker/datepicker.ts

```diff
@@ -459,21 +460,14 @@ export class SbbDatepickerElement extends LitElement {
    * @internal
    * Returns current date or configured date.
    */
-  public now(): Date {
-    if (this._hasDataNow()) {
-      const today = new Date(readDataNow(this));
-      today.setHours(0, 0, 0, 0);
-      return today;
-    }
-    return this._dateAdapter.today();
-  }
-
-  private _hasDataNow(): boolean {
-    return this.hasAttribute('data-now');
+  public getNow(): Date {
+    const today = new Date(this.dateNow);
+    today.setHours(0, 0, 0, 0);
```

Comment:
As there is often the case of using a Date without time information, I think this could also be a getter in the new mixin